tests/rbd/rbd_migration_ec2ec_dafault_k_m.py

```diff
@@ -0,0 +1,110 @@
+Test Case Flow -
+1. Create EC pool with default k_m values(2,2) and an image
+2. Create an another EC pool with default k_m values(2,2) for destination pool
+3. Mount the images and run io from clients
+4. Stop IO and prepare migration process from EC pool to EC pool
+5. Start IO from client on destination images
+6. Execute migration
+7. Commit migration and check the destination image info
+"""
+
+from tests.rbd.exceptions import RbdBaseException
+from tests.rbd.rbd_utils import (
+    Rbd,
+    initial_rbd_config,
+    verify_migration,
+    verify_migration_commit,
+)
+from utility.log import Log
+from utility.utils import run_fio
+
+log = Log(__name__)
+
+
+def run(**kw):
+    """verify live migration of images from ec pool to ec with dafault k_m values.
+    Args:
+        kw: Key/value pairs of configuration information to be used in the test
+    Returns:
+        int: The return value. 0 for success, 1 otherwise
+    """
+    log.info("Starting execution of live migration from EC pool to EC pool")
+    rbd = Rbd(**kw)
+    config = {
+        "ec-pool-only": True,
+        "ec-pool-k-m": "2,2",
+        "crush-failure-domain": "osd",
+        "ec_pool_config": {
+            "pool": "rbd_ECpool1",
+            "image": "rbd_image_ec1",
+            "data_pool": "data_pool_ec1",
+            "size": "10G",
+        },
+    }
+    kw["config"].update(config)
+    log.info(kw["config"])
+    initial_rbd_config(**kw)
+    image = f'{config["ec_pool_config"]["image"]}'
+    ecpool1 = f'{config["ec_pool_config"]["pool"]}'
+    src_spec = ecpool1 + "/" + image
+
+    config = {
+        "ec-pool-only": True,
+        "ec-pool-k-m": "2,2",
+        "do_not_create_image": True,
+        "crush-failure-domain": "osd",
+        "ec_pool_config": {
+            "pool": "rbd_ECpool2",
+            "data_pool": "data_pool_ec2",
+        },
+    }
+    kw["config"].update(config)
+    log.info(kw["config"])
+    initial_rbd_config(**kw)
+    ecpool2 = f'{config["ec_pool_config"]["pool"]}'
+    dest_spec = ecpool2 + "/" + image
+
+    try:
+        client = kw["ceph_cluster"].get_nodes(role="client")[0]
+        run_fio(image_name=image, pool_name=ecpool1, client_node=client)
+
+        # Perpare the migration process from EC pool to destination EC pool
+        rbd.migration_prepare(src_spec, dest_spec)
+
+        # verify migration prepare
+        verify_migration(rbd, dest_spec)
+
+        # run io's on the destination image
+        run_fio(image_name=image, pool_name=ecpool2, client_node=client)
+
+        # Execute migration process
+        rbd.migration_action(action="execute", dest_spec=dest_spec)
+
+        # commit migration process
+        rbd.migration_action(action="commit", dest_spec=dest_spec)
+
+        # verify commit migration
+        verify_migration_commit(rbd, ecpool2, image)
+
+        # Display image info and verify
+        image_info = rbd.image_info(ecpool2, image)
+        log.info(f"Image info : {image_info}")
```

Comment:
[Optional]
I think image info can be pushed to debug

Reply:
Done.